vercel/turborepo
# examples: Update `basic` example to Next.js 15.

### Description

As the title says, updating to Next.js 15 in `basic` so `create-turbo` gives people the latest and greatest.

I've also taken the time to:
- Make the content of the applications Turborepo
- Added a `check-types` task for folks who are interested in running TypeScript checks.

### Testing Instructions

Our examples CI should make sure things are still building correctly. Would appreciate if someone could open it up in their editor and see if they get any squiggly lines, as well!
